### PR TITLE
Avoid throwing exceptions from native modules when the host activity …

### PR DESCRIPTION
**Pull request by mganandraj**
#### Please select one of the following
- [ ] I am removing an existing difference between facebook/react-native and microsoft/react-native :thumbsup:
- [ ] I am cherry-picking a change from Facebook's react-native into microsoft/react-native :thumbsup:
- [ ] I am making a fix / change for the macOS implementation of react-native
- [x] I am making a change required for Microsoft usage of react-native

(give an overview)

Office apps are not based on AppCompat/Fragment activities. This change avoid RN throwing exceptions when the host activities are not FragmentActivity.

I'm working on pushing this change to upstream Facebook repo too.

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native/pull/201)

### CODE REVIEW

**Comment by mganandraj** [comment]
Raised a PR in the upstream FB repo : https://github.com/facebook/react-native/pull/27425

File: Libraries/Renderer/shims/ReactFeatureFlags.js~d9a25c2d04f32259d711e38195689d376d4a06a7 [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+'use strict';
+
+const ReactFeatureFlags = {
+  debugRenderPhaseSideEffects: false,
+  debugRenderPhaseSideEffectsForStrictMode: false,
+  warnAboutDeprecatedLifecycles: false,
+};
+
+module.exports = ReactFeatureFlags;
```
Comment (mganandraj) [inline Libraries/Renderer/shims/ReactFeatureFlags.js~d9a25c2d04f32259d711e38195689d376d4a06a7]: This file is added unintentionally. Will be removed in a future commit.

**Comment by acoates-ms** [comment]
Unless its very urgent, can we hold off on merging until the upstream PR gets merged, in case there is core review feedback from Facebook on that one.

**Comment by mganandraj** [comment]
> Unless its very urgent, can we hold off on merging until the upstream PR gets merged, in case there is core review feedback from Facebook on that one.

Sure.. we can wait.

**Pull request merged**